### PR TITLE
Third time!, third time!

### LINKED ISSUE #8137: Whatever, this is mi third time doing this, pfffff
Hello! :wave: Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by poppygetstrash**
Adding tepa to the map
closes #8137

### CODE REVIEW

File: _pins/poppygetstrash.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+githubHadle: poppygetstrash
+latitude: 20.659699
+longitude: -103.349609
```
Comment (davidnhz) [inline _pins/poppygetstrash.json]: You need to add `---`  at the end of the file.

**Comment by poppygetstrash** [comment]
adding 
closes #8141

File: _pins/poppygetstrash.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+githubHadle: poppygetstrash
```
Comment (crichID) [inline _pins/poppygetstrash.json]: @poppygetstrash you will also want to correct the spelling of `githubHandle` :wink:

Comment (poppygetstrash) [inline _pins/poppygetstrash.json]: thank you!

**Comment by crichID** [comment]
@poppygetstrash your code looks perfect! Please go ahead and click the big green button below to **Merge pull request**. In an effort to keep things in this repository clean, I will merge this if it is still open on July 7th.

**Pull request merged**